src/Components/Wrapper/Wrapper.tsx

```diff
@@ -100,19 +100,23 @@ const Wrapper = (props: PropsWithChildren<{}>) => {
       setTranslations(value);
     });
   }, []);
-  let defaultLanguage = (localStorage.getItem('language') ?? 'en-us') as Language;
+  const defaultLanguage = (localStorage.getItem('language') ?? 'en-us') as Language;
+
+  let userLanguage = navigator.language.toLowerCase() as Language;
+  userLanguage = languageOptions[userLanguage] ? userLanguage : defaultLanguage;
```

Comment:
We want to set the language to the `localStorage` language if it exists, and if it does not exist, we want to set the language to the browser default. This is because we do not want to change the language if the user refreshes the page. Also to check if the navigator language exists, we want to do something like what is shown bellow to account for region codes:

```
Object.keys(languageOptions).some((lang) => userLanguage.slice(2) === lang.slice(2))
```

Reply:
Thanks for clarification. Now, we check the `localStorage` for the preferred language. If it's not specified, we then check the user's browser language. If none of them is available, we set the default language to `en-us`.